docs/docs/04-concepts/30-advanced/05-workflow-lifecycle.md

```diff
@@ -0,0 +1,10 @@
+# Workflow Lifecycle
```

Comment:
Don't forget about these pages. Maybe we could add an 'UNDER CONSTRUCTION' flag to any page that is in progress.